wordpress-mobile/WordPress-iOS
# Update release to 1.31.1

Update Gutenberg to release 1.31.1

To test:
 - Smoke test the GB writing flow
 - Check the quote block  in dark mode and see that the vertical bar shows up correctly
 - Check the buttons block and see if adding multiple buttons and press Enter works correctly.

PR submission checklist:
